src/test/java/sandbox/Sandbox.java

```diff
@@ -122,7 +156,16 @@ private void runScript(String url, String scriptName, String scriptType) throws
                 throw new IllegalArgumentException("Unsupported script type: " + scriptType);
         }
 
-        driver.get(url);
+        do {
+            try {
+                driver.get(url);
+                break;
+            }
+            catch (Exception e){
```

Comment:
IDE was not letting me to put "SocketTimeoutException". So, I just added this. Please check it.